stylesheet.css

```diff
@@ -316,6 +317,9 @@ a.share-button {
 
 .navbar {
```

Comment:
Making the `.navbar` fixed results in the rest of the page being shifted up by the height of the `.navbar` (see below for an illustration of the problem). Fixing this requires some creativity but I think the easiest way to solve it is to update the padding on the `.hero` element (as I did below). What do @simple-icons/maintainers think? 

| Current | Changes |
|---|---|
| ![current](https://user-images.githubusercontent.com/3742559/68162453-ba5b4e00-ff60-11e9-982e-3199e923edf3.png) | ![changes](https://user-images.githubusercontent.com/3742559/68162454-bc251180-ff60-11e9-86a7-0dc93bfd7b9e.png) |

I think the snippet below fits best with the current layout (you can just copy the whole thing and replace it with the existing styling for `.hero`):

```css
.hero {
  font-size: 1.5rem;
  line-height: 2rem;
  margin: 0;
  padding: 7.5rem 1.5rem 1.5rem;
  text-align: center;
}
@media (min-width: 45rem) {
  .hero {
    padding: 9rem 3rem 1.5rem;
  }
}
```